### PR TITLE
add render list

### PR DESCRIPTION
**Pull request by dagmawiii03**
**project requirements**

- Add a new JavaScript file and import it as a module:
- it will contain methods related to the status updates (completed: true / false).
- Add event listener to the checkbox (change).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button (use filter() method).
- Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved).

### CODE REVIEW

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @dagmawiii03, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

-  Descriptive Pull Request title and summary. ✔️
- Add event listener on the checkbox. ✔️
- Task object's values for completed is updated upon user actions. ✔️
- Function for the "Clear all completed" button. ✔️
- The updated array of items is stored in local storage. ✔️
- Good use of Github flow ✔️

### Changes Required

- [ ] I would kindly suggest you work on MIT License link as currently the link is broken and its throwing an `error 404`

MIT 👇

mit

- [x] Also the `live demo` is not linking to the deployed project kindly remove the heading if you didn't deployed the project yet but if you have the deployed link, kindly add it


- [x]  _Please check the comments under the review._

## Optional suggestions

[OPTIONAL]

- [x] In the `About project` heading of your Readme file, the description about the project is not talking about the current project, I notice you are using the description about the capstone project in `module 1` for this project. Kindly please work on that

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/addRemove.js [hunk 1]
```diff
@@ -22,54 +21,17 @@ export const removeTask = (currentItem) => {
...
-  clearRemove.forEach((cl) => cl.remove());
+export const add = (item) => {
+  // const clearRemove = document.querySelectorAll('.clear');
+  // clearRemove.forEach((cl) => cl.remove());
   todoList.push({
     description: item,
     completed: false,
     index: todoList.length + 1,
   });
   localStorage.setItem('todoList', JSON.stringify(todoList));
```
[the remark this answers is not in the document]
Comment (brainconnect93) [reply src/addRemove.js:36]: @dagmawiii03, please you shouldn't work on your project after you have submitted for review, You are working on your project and pushing to the t PR after you have submitted for reviewer. You should always exercise patience after you have submitted for review and try as much as you can to wait for the reviewer to drop his/her comment.
Thanks

Comment (dagmawiii03) [inline src/addRemove.js]: Okay and thanks for the comments

**Review by Whoistolu — COMMENTED** [review]
Hi @dagmawiii03 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project, that does not have the proper project description. Your project has a description for module 1. Kindly fix this to make your project more professional. The project description should be about To Do list: an interactive list not a mobile design for an online website for a conference. Also, kindly remove the broken live demo and deployment link on the README as they are broken and not a requirement for this milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/addRemove.js (hunk 1 above)
Comment (brainconnect93) [reply src/addRemove.js]: @dagmawiii03, please you shouldn't work on your project after you have submitted for review, You are working on your project and pushing to the t PR after you have submitted for reviewer. You should always exercise patience after you have submitted for review and try as much as you can to wait for the reviewer to drop his/her comment.
Thanks

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @dagmawiii03 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights

All linters are passing ✔️
Good commit history ✔️
Good PR description ✔️
Follow Gitflow ✔️


 ## Required Changes ♻️
_Check the comments under the review._

- [x] As suggested previously the live demo is not linking to the deployed project kindly remove the heading if you didn't deploy the project yet but if you have the deployed link, kindly add it.
- [ ] Please as suggested this project does not have the proper project description. Your project has a description for module 1. Kindly fix this to make your project more professional. The project description should be about To Do list: an interactive list not a mobile design for an online website for a conference. Also, kindly remove the broken live demo and deployment links on the README as they are broken and not a requirement for this milestone.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — APPROVED** [review]
Hi @dagmawiii03 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights

All linters are passing ✔️
Good commit history ✔️
Good PR description ✔️
Follow Gitflow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**